### ISSUE TITLE
dotted line if tooltip is added

### ISSUE THREAD

**Issue by Kugeleis** [picocss/pico#629]
### Current Behavior
```js
        <ul>
            <li>
                <label data-tooltip="Toggle dark and light mode" for="theme-toggle" class="theme-toggle-label">{%
                    include 'flaticons/bright.svg' %}
                    <input name="terms" class="theme-toggle" id="theme-toggle" type="checkbox" role="switch"
                        checked />{%
                    include 'flaticons/moon.svg' %}
                </label>
            </li>
        </ul>
```
produces a dotted line below the element.
`data-tooltip="Tooltip"` within the <input> element destroys formatting completely.

### Expected Behavior
Only tooltip added

**Comment by Yohn** [comment]
Are you using the newest version of Pico here? Cause https://github.com/picocss/pico/blob/6dc6489e69cc1f22daa938ebd38c02400032428c/scss/components/_tooltip.scss#L12-L16 prevents the dotted line on inputs in the current release of Pico here.

**Comment by Kugeleis** [comment]
Yes, I'm using the latest version (2.0.6). Following your hint I added `style="border-bottom: none;"` to the element locally as a patch.

**Comment by Yohn** [comment]
I was able to duplicate your issue, and found a fix which I included in my fork of [PicoCSS](/Yohn/PicoCSS).

The main change for your issue is to update that line I pointed out previously to the follow:
`&:not(:has(a, button, input, [role="button"])):not(a, button, input, [role="button"]) {`

**Comment by boustanihani** [comment]
Please also remove the dotted line if a tooltip is added to a `<select>` element.

**Comment by mariusGundersen** [comment]
I have stumbled on this issue too, and believe nothing should have dotted line, except maybe `<span>`
